README.md

```diff
@@ -16,7 +16,7 @@ Feel free to [create an issue](https://github.com/EddieJaoudeCommunity/EddieBot/
 
 - Timezone, listens for messages that contain 1:30pm UTC and replies with common timezones translation
 
-- Code of Conduct
+- Code of Conduct(COC)
```

Comment:
I think it would be better with a space before the opening `(`

Reply:
Okay, let me fix that.